set.mm

```diff
@@ -20984,23 +20981,20 @@ requires either a disjoint variable condition ( ~ sb5 ) or a non-freeness
 
   ${
     $d x y $.
-    $( Version of ~ equsb1 with a disjoint variable condition, which neither
-       requires ~ ax-12 nor ~ ax-13 .  (Contributed by BJ, 11-Sep-2019.)
-       Remove dependencies on axioms.  (Revised by Wolf Lammen, 30-May-2023.)
-       (Proof shortened by Steven Nguyen, 19-Jun-2023.)
+    $( Obsolete version of ~ equsb1v as of 19-Jun-2023.  (Contributed by BJ,
```

Comment:
I checked out set.mm as of 10-Jul-23, and equsb1v looked the same there as the OLD version here.  I think it became obsolete on 11-Jul.

Reply:
Thanks. This is corrected.

Reply:
rewrapping is needed.

Reply:
Rewrapped. I changed the order of two paragraphs in the comment of df-sb in order to have the two paragraphs about notation next to each other.